### PR TITLE
Correct issues where dates were being set inadvertantly

### PR DESCRIPTION
**Pull request by Camwyn**
PHP: stop setting a default date on the date/time pickers.
CSS fixes for cropped tooltips. This changes the transition for the panels.
        Make "No active fields. Add new fields or" smaller.

Ticket: https://central.tri.be/issues/80627

Required: https://github.com/moderntribe/event-tickets-plus/pull/390

### CODE REVIEW

File: src/admin-views/edit_admin_panel.php [hunk 1]
```diff
@@ -111,7 +105,7 @@ class="ticket_form_title tribe-dependent"
 						<label class="ticket_form_label ticket_form_left" for="ticket_description"><?php esc_html_e( 'Description:', 'event-tickets' ); ?></label>
 						<textarea rows="5" cols="40" name="ticket_description" class="ticket_field ticket_form_right" id="ticket_description"></textarea>
 						<div class="input_block">
-							<label class="tribe_soft_note"><input type="checkbox" id="tribe_tickets_show_description" name="ticket_show_description" value="1" class="ticket_form_left" checked> Show description on front end and emailed tickets.</label>
+							<label class="tribe_soft_note"><input type="checkbox" id="tribe_tickets_show_description" name="ticket_show_description" value="1" class="ticket_field ticket_form_left" checked> Show description on front end and emailed tickets.</label>
```
Comment (ggwicz) [inline src/admin-views/edit_admin_panel.php]: Make translatable with `esc_html_e()`

File: src/resources/postcss/tickets-refresh.pcss [hunk 2]
```diff
@@ -54,12 +53,20 @@
...
-	transition: .5s cubic-bezier(.55, .085, .68, .53);
+	visibility: visible;
 	width: 100%;
 	will-change: transform;
 
+	&[aria-hidden='true'] {
+		opacity: 0;
+		position: absolute;
+		visibility: hidden;
+		right: -100%;
```
Comment (ggwicz) [inline src/resources/postcss/tickets-refresh.pcss]: alphabetize the rules (just switch `visibility:` with `right:`)

File: src/resources/js/tickets.js [hunk 3]
```diff
@@ -64,6 +65,224 @@ var ticketHeaderImage = window.ticketHeaderImage || {};
 		}
 	};
 
+	function format_date(date) {
```
Comment (bordoni) [inline src/resources/js/tickets.js]: Can we replace these two methods (time management) by using moment.js? It's been used already. (I think)

Comment (Camwyn) [reply src/resources/js/tickets.js]: Oh, gawd yes - `moment.js` would be wonderful 😉

**Comment by Camwyn** [comment]
This now uses `moment.js` for formatting 🎉 
I've added some comments about the implementation, as I initially got a moment deprecation warning. Open to suggestions on my approach to avoiding that.

**Comment by Camwyn** [comment]
Hey @ggwicz - no, I'm using 4. There is more nesting than I'd like in some spots, but it's all 4-space tabs.
Maybe you're looking at the end of an ajax request?
```
$.post(
	ajaxurl,
	params,
	function( response ) {
		// stuff
	} ).complete( function( response ) {
		// more stuff
	} );
```
The end of `$.post` doesn't line up there because of the `} )`? Wasn't sure that dropping the `);` to a new line was worth it?

Or point out an example for me - I may be blind 😉

**Comment by ggwicz** [comment]
Hey @Camwyn, I apologize, I was misreading the spacing as shown in GitHub's code viewing UI. Your spacing is fine; ignore my comment.

**Pull request merged**